.github/workflows/docker-publish.yml

```diff
@@ -75,9 +89,18 @@ jobs:
           # dynamically set date as a suffix
           tags: |
             type=ref,event=branch,suffix=-{{date 'YYYYMMDDHHmm'}}
-
-      - name: build
-        run: make build-components
+            type=ref,event=branch
+          labels: |
+            maintainer="Nervos Core Dev <dev@nervos.org>"
+            org.opencontainers.image.authors="Nervos Core Dev <dev@nervos.org>"
+            source.component.Godwoken="https://github.com/nervosnetwork/godwoken"
+            source.component.Godwoken-Scripts="https://github.com/nervosnetwork/godwoken-scripts"
+            source.component.Polyjuice="https://github.com/nervosnetwork/godwoken-polyjuice"
+            source.component.omni_lock="https://github.com/nervosnetwork/ckb-production-scripts"
+            ref.component.Godwoken=${{ steps.build.outputs.GODWOKEN_REF }}
+            ref.component.Godwoken-scripts=${{ steps.build.outputs.GODWOKEN_SCRIPTS_REF }}
+            ref.component.Polyjuice=${{ steps.build.outputs.POLYJUICE_REF }}
+            ref.component.omni_lock=${{ steps.build.outputs.OMNI_LOCK_REF }}
```

Comment:
Can we use the original repo name as the key? `source.component.godwoken-polyjuice`

Reply:
"CKB-production-scripts" should be labeled instead of "omni_lock".
From the docker labels we want to know is that the git repository and git reference. Please keep it original and sample.